### PR TITLE
Jchen/add providers to retrieveDeviceSerializer

### PR DESCRIPTION
**Pull request by JohnnyChen765**

### CODE REVIEW

File: mds/apis/prv_api/vehicles.py [hunk 1]
```diff
@@ -89,10 +91,14 @@ class Meta:
         )
 
 
-class RetrieveDeviceSerializer(BaseDeviceSerializer):
+class RetrieveDeviceSerializer(DeviceSerializer):
     areas = serializers.SerializerMethodField()
+    provider_logo = serializers.CharField(
+        source="provider.logo_b64",
+        help_text="logo of the service provider of the device",
```
Comment (lionel-panhaleux) [inline mds/apis/prv_api/vehicles.py]: maybe state the format (base64) in the help_text ?

File: mds/apis/prv_api/vehicles.py [hunk 2]
```diff
@@ -89,10 +91,14 @@ class Meta:
...
+    )
 
-    def get_areas(self, obj):
+    def get_areas(self, obj) -> List[Dict[str, str]]:
```
Comment (lionel-panhaleux) [inline mds/apis/prv_api/vehicles.py]: Does this help with schema generation ? If yes, that's good to know

Comment (JohnnyChen765) [reply mds/apis/prv_api/vehicles.py]: yes, it does.

Comment (JohnnyChen765) [reply mds/apis/prv_api/vehicles.py]: It seems to be an alternative solution to swagger decoration

File: CHANGELOG.rst [hunk 3]
```diff
@@ -19,6 +19,7 @@ Changelog
 
 - Fix conversion of datetime objects to MDS timestamps in the APIs.
 - Also take into account "battery_charged" event type from providers.
+- Added areas methodField to DeviceSerializer
```
Comment (cedricco) [inline CHANGELOG.rst]: To remove here. A merge from master after a release put your entry in both version.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,8 +5,8 @@ Changelog
 ------------------
 
 - Fix conversion of datetime objects to MDS timestamps in the APIs.
-- Created separate RetrieveDeviceSerializer with areas methodField
-
+- Created separate RetrieveDeviceSerializer with areas methodField.
+- Added provider_logo in RetrieveDeviceSerializer.
```
Comment (cedricco) [inline CHANGELOG.rst]: We use imperative form as a convention: 
"add provider_logo [...]"

**Pull request merged**